src/components/SingleReview.jsx

```diff
@@ -32,9 +45,9 @@ export const SingleReview = ()=> {
             <p>Owner: {review.owner}</p>
             <p>Designed by: {review.designer}</p>
             <p>Category: {review.category}</p>
-            <p>Votes:{review.votes}</p>
-            <button>👍</button>
-            <button>👎</button>
+            <p>Votes:{review.votes + voteChange}</p>
```

Comment:
This way deffinitely works, I'd probably recommend updating the review.votes optimisticly and using the response from the patch request to confirm this rarther than using a seperate piece of state however it's completely up to you how you want to implement this.